nss-evening-cohort-24/gitsub-team-3
# overview-styling-changes

## Description
- Reformatted repos cards - font color, added primary codebase
- Integrated changes from previous PR that restructured pinned repo card widths
- Added new div, html.index, line 26 for just the 'Pinned' header. 
```
<div id="pinned-repos-header"></div>
```
- Dynamically rendered 'Pinned' heading to avoid styling issues with flex-box container. 
- Reformatted with Prettier for readability

## Related Issue
#15 

## Motivation and Context
This change integrates team made changes to container sizing and makes page stylistically appealing to the user. Adds cohesion of styles across site. 

## How Can This Be Tested?
Stand a live server and view changes by adding repos through the 'Create a Pinned Repo' form. Cards should be added with uniform sizing. 

## Screenshots (if appropriate):

## Types of changes
